### ISSUE TITLE
Can not get the close button label to align properly.

### ISSUE THREAD

**Issue by martinduo** [jquense/react-bootstrap-modal#37]
The label sits on the bottom edge of the close button.
I have tried adding my own styling to the button label (negative margin, etc) but nothing worked for me. See the image for the problem.  (I tried to attach the image, but only get a link to github. Does this work for you?)
![capture](https://cloud.githubusercontent.com/assets/879852/22620735/3b424764-eae0-11e6-9a71-61b23f683b97.PNG)

**Comment by jquense** [comment]
hi there. I can see the picture. I'm not sure what might be wrong but I don't think it's due to the css included here, which doesn't have any styling for buttons. it might be a conflict with your css and bootstrap

**Comment by martinduo** [comment]
Thanks for the quick reply. Since you confirmed that the problem was not likely due to your css, I went back and dug deeper into my css. I'm using react-bootstrap with the bootstrap-cerulean.css, so lots to look at. I did manage to get the layout I wanted by adding more local css of my own for the Modal.Footer, Modal.Dismiss, and the footer's button component in your component. Thanks for your answer and your work on the Modal code.
